deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -482,7 +547,7 @@ spec:
             - name: ES_PASSWORD
               value: ""
             - name: ES_HOST
-              value: ""
+              value: "elastic-package-stack_elasticsearch_1"
```

Comment:
Was this change intentional ? This is a proposed manifest and I don't think `elastic-package-stack_elasticsearch_1` is of any value to a customer.

Reply:
I've already removed it - it was used to test locally